text/0000-transaction-bundle/0000-transaction-bundle.md

```diff
@@ -0,0 +1,572 @@
++ Feature name: (fill me in with a unique ident, `my_awesome_feature`)
++ Start date: 2019-09-06
++ RFC PR: [iotaledger/bee-rfcs#3](https://github.com/iotaledger/bee-rfcs/pull/3),
+[iotaledger/bee-rfcs#18](https://github.com/iotaledger/bee-rfcs/pull/18),
+[iotaledger/bee-rfcs#20](https://github.com/iotaledger/bee-rfcs/pull/20)
++ Bee issue: [iotaledger/bee#0000](https://github.com/iotaledger/bee/issues/0000)
+
+# Summary
+
+## Transaction
+
+In the context of IOTA, a `Bundle` is a message that is shared over the network, and a `Transaction` is the fundamental
+unit that a `Bundle` is constructed from. To use TCP as a well known analogy, a `Bundle` corresponds to a message, and a `Transaction` corresponds to a packet.
```

Comment:
We should call `Bundle` a `Stream` instead.

Reply:
I wouldn't introduce new terms in this RFC for bundles/transactions, we should stick to them as we are used to. Introducing new, better terms could be done with Coordicide.